DNN Platform/Library/Entities/Modules/Settings/SettingsRepository.cs

```diff
@@ -169,7 +176,12 @@ private T Load(CacheItemArgs args)
                 }
                 else if (attribute is PortalSettingAttribute && portalId != -1 && PortalController.Instance.GetPortalSettings(portalId).ContainsKey(mapping.FullParameterName))
                 {
-                    settingValue = PortalController.Instance.GetPortalSettings(portalId)[mapping.FullParameterName];
+                    var psa = (PortalSettingAttribute)attribute;
+                    settingValue = PortalController.Instance.GetPortalSettings(portalId, string.Empty)[mapping.FullParameterName];

```

Comment:
why the introduction of the `string.Empty` argument here?

Reply:
I was working on localization but pulled out of it. I'll reset this.